src/odemis/acq/stream/_sync.py

```diff
@@ -1953,6 +1961,20 @@ def _runAcquisition(self, future):
             logging.debug("Starting e-beam sync acquisition with components %s",
                           ", ".join(s._detector.name for s in self._streams))
 
+            # initialize the EBIC scan controller and verify it is in ready state
+            ebic_det = None
+            try:
+                ebic_det = model.getComponent(role='ebic-detector')
+            except LookupError:
+                logging.debug("No EBIC detector found skipping synchronized acquisition of EBIC.")
+
+            if ebic_det:
+                if ebic_det.scanState.value != ephemeron.STATE_NAME_IDLE:
+                    logging.warning(f"Scan state of the EBIC scan controller is not {ephemeron.STATE_NAME_IDLE}")
+                else:
+                    # Set the EBIC scan controller state to running
+                    ebic_det.scanState.value = ephemeron.STATE_NAME_RUNNING
```

Comment:
Is this to start the scanning? If so, this should be done by following the standard Odemis API. It consist in subscribing to the detector's DataFlow. The function already does that though... so normally there is nothing really special.